README.md

```diff
@@ -1,34 +1,166 @@
+the design should be able to independently create visually-identical implementations
+of the specification without having to ask the original designer for clarifications.
+They should also be flexible enough that they can be adapted to a wide variety of use cases
+and design considerations.
+
+The specification should explain:
+
+1. Underlying Design principles
 2. The Why, The What, and The How
 3. Fonts
-* Font styles. This means font family, weight, size, font style, font stack (including fall backs)
-* Font styles should be concise. We encourage only a few per font styles and we highly encourage using the same font styles.
+    * Font styles. This means font family, weight, size, font style, font stack (including fall backs)
+    * Font styles should be concise. We encourage only a few per font styles and we highly encourage using the same font styles.
 4. Breakpoints
-* And differences at those breakpoints
+    * And differences at those breakpoints
 5. Colors
-* Different shades of each, when and where to each, and what colors of text are allowed with each.
-* Sizing & Measurements
-* Present measurements with consideration to the CSS box model.
-* When appropriate, the measurement should or can be a percentage. Please be aware of trying to make designs fluid.
-* Please be comfortable with using and understanding flexbox.
+    * Different shades of each, when and where to each, and what colors of text are allowed with each.
+    * Sizing & Measurements
+    * Present measurements with consideration to the CSS box model.
+    * When appropriate, the measurement should or can be a percentage. Please be aware of trying to make designs fluid.
+    * Please be comfortable with using and understanding flexbox.
 6. Padding & Margin
-* Please understand the difference between padding and margin.
-* Please understand the CSS box model. We need this to be considered when specifying padding and margin.
+    * Please understand the difference between padding and margin.
+    * Please understand the CSS box model. We need this to be considered when specifying padding and margin.
 7. Image Formats (if applicable to the design)
-* Specs should include sizes, and image ratios. 
-* Consider how the image handles at different breakpoints. 
-* What image types are appropriate
-* Please use svg’s whenever possible.  This means anything but a photo should be SVG. SVG is Scalable vector graphic. It performs well at small and large and is a smaller file.
-* Photos should be jpg or png.
+    * Specs should include sizes, and image ratios. 
+    * Consider how the image handles at different breakpoints. 
+    * What image types are appropriate
+    * Please use svg files whenever possible.  This means anything but a photo should be SVG. SVG is Scalable vector graphic. It performs well at small and large and is a smaller file.
+    * Photos should be jpg or png.
 8. Use Cases & Corner cases
-* The specs should specify how the design handles common and unique cases.
```

Comment:
I still think we should have a section mentioning how to add your component's section to this repo - at least mention the Examples folder and how it can be copied and used as a template to start adding a new project.